### ISSUE TITLE
MacBook Pro: Add touchbar customization (emojis...)

### ISSUE THREAD

**Issue by pabloegpf1** [meetfranz/franz#550]
### Expected Behavior

Provide custom options for touchbar in MacBook pro new gen so that we can use emojis without using the trackpad in services such as Whatsapp and others.
Also there are several extra options it could be used to in the future (private mode, dark mode...).
Other apps for Whatsapp already use this feature (ChatMate for Whatsapp)
### Current Behavior

Currently the touchbar is only used for moving from one service to other, it has much more potential.
### Context

The official Whatsapp desktop app doesn't support the touchbar customization, I am looking forward to see this feature in this app.
### Your Environment

* Franz Version used: 5.0
* Operating System and version: MacOs High Sierra

**Comment by adlk** [comment]
The TouchBar support hasn't received much love yet. A few improvments, this included, are on our todo list.

**Comment by pabloegpf1** [comment]
Any ETA?

Thanks for the response

**Comment by stevenvw1973** [comment]
Any news yet on this? Touchbar support is starting to grow on various applications now.

**Comment by BorBorBor** [comment]
I commented on #576 before noticing this was a duplicate. But to be honest I'm not expecting any 'special' behavior or complex customizations: the basic functionality is that it should behave on the text field like it does on most text fields. So text prediction and emoji button should work and currently they are being overridden by a 'tab bar'.

**Comment by julianmenzler** [comment]
Any update on this one?

**Comment by omarf1mail** [comment]
Hi everyone
just discovering Franz before considering any subscription I trying it out and miss very much the icons on the touch bar that I got used in the native Apps on my mac
any chance to offer to replace the current services names and map the native app touch bar instead ?

thank you very much

**Comment by mustafaozhan** [comment]
Yes Exactly, there is no need for services in touch-bar they have already shortcuts with Command 1,2,3,4...

This is the only reason makes Franz behind of other services, I personally want to use 1 app for every services but because it is not comfortable to chat without emoji i use different apps for WhatsApp and Messenger.

**Comment by brunocramos** [comment]
+1 for this, I decided to start using Franz back after a few years, but lacking this feature makes me reconsider it. Any updates/expectations for this feature?

**Comment by downinthesewer** [comment]
Any update on this? Just implement the default behaviour of the Touch Bar when in a text field. I would sign up for a paid subscription but not being able to hit the emoji button is a dealbreaker.
